src/core/model_config.proto

```diff
@@ -1144,6 +1144,25 @@ message ModelOperations
   repeated string op_library_filename = 1;
 }
 
+//@@
+//@@ .. cpp:var:: message ModelTransactionType
+//@@
+//@@    The specification that describes the nature of transactions
+//@@    to be expected from the model.
+//@@
+message ModelTransactionType
+{
+  //@@  .. cpp:var:: bool decoupled
+  //@@
+  //@@     Indicates whether responses generated by the model are decoupled with
+  //@@     the requests issued to it, which means the number of responses
+  //@@     generated by model may differ from number of requests issued. If
```

Comment:
".. of requests issued, and that the responses may be out of order relative to the order of requests."